proposals/semi-auto-properties.md

```diff
@@ -52,6 +52,13 @@ The following changes are to be made to [classes.md](https://github.com/dotnet/c
 
 ...
 
+```diff
+- If the auto-property has no set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
++ If the auto-property has semicolon-only get accessor without a set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a semicolon-getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
```

Comment:
I'm not sure if "semicolon-getter-only" is clear enough.

Reply:
It is to me, or it could be `an auto property which only has a semicolon getter`. I'd nitpick:

- has *a* semicolon-only get accessor
- `auto property` not `auto-property` (I know, this was original)

Reply:
I wonder if the spec should say something about getter-only properties being assignable in the body of a constructor even when the getter is not semicolon-only and therefore the backing field is not readonly.

Reply:
Yup I just noticed an overlap and a possibility to divide this into two separate sentences depending on what the initializer behavior would be. Let me mark as a draft for now as it doesn't seem to be accurate.

Reply:
I think the wording is more accurate now?

Reply:
Lgtm!